content/docs/components/hyperparameter.md

```diff
@@ -4,25 +4,91 @@ description = "Using Katib to tune your model's hyperparameters"
 weight = 5
 +++
 
-## Using Katib
+Hyperparameter Tuning on Kubernetes.
+This project is inspired by [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf). Katib is a scalable and flexible hyperparameter tuning framework and is tightly integrated with kubernetes. Also it does not depend on a specific Deep Learning framework (e.g. TensorFlow, MXNet, and PyTorch).
```

Comment:
Let's standardise the capitalization, so that we only capitalize brand names. That practice makes docs easier to read, and makes sure people know which things are branded products.

s/Tuning/tuning
s/kubernetes/Kubernetes
s/Deep Learning/deep learning